resources/views/projects/tasks/create.blade.php

```diff
@@ -0,0 +1,83 @@
+@extends('layouts.app')
+
+@section('title', 'New task')
+
+@section('content')
+<div class="padded content">
+	<h1>New task</h1>
+	<!-- TODO: action url -->
+	<form class="ui form {{ $errors->any() ? 'error': '' }}" method="post" enctype="multipart/form-data" action="">
+		@csrf
+		<div class="ui stackable grid">
+			<div class="ten wide column" style="padding-left: 0 !important;">
+				<!-- Title -->
+				<div class="field {{ $errors->has('title') ? 'error': '' }}">
+					<label for="title">Title</label>
+					<input type="text" id="title" name="title" placeholder="e.g. Design UI layout" value="{{ old('title') }}">
+				</div>
+				<!-- Description -->
+				<div class="field {{ $errors->has('description') ? 'error': '' }}">
+					<label for="description">Description</label>
+					<textarea id="description" name="description" placeholder="e.g. Design the layout for the application in all devices...">{{ old('description') }}</textarea>
+				</div>
+			</div>
+			<div class="six wide column" style="padding-left: 0 !important;">
```

Comment:
No need for the `important` here

Reply:
Semantic usa `padding-left : 1rem !important` para las columns. 🙁

Reply:
Semantic usa padding-left: `1rem !important` en todos los columns.  Metí todo en columnas de forma que quede alineado y siga un layout (I think). Let me know if it makes sense.